quinn-proto/src/connection/mod.rs

```diff
@@ -3127,6 +3131,23 @@ impl Connection {
         is_1rtt: bool,
     ) {
         self.total_authed_packets += 1;
+        if let Some(last_allowed_receive) = self.abandoned_paths.get(&path_id) {
+            if now > *last_allowed_receive {
+                // The peer failed to respond with a PATH_ABANDON in time.
+                warn!("missing PATH_ABANDON from peer");
+                if !self.state.is_closed() {
+                    // TODO(flub): What should the error code be?
+                    self.state.move_to_closed(TransportError::NO_ERROR(
+                        "peer ignored PATH_ABANDON frame",
+                    ));
+                    self.close_common();
+                    self.set_close_timer(now);
+                    self.close = true;
+                }
+                return;
+            }
+        }
```

Comment:
Not sure about the position of this. Perhaps this should move to the end of the of the function and not do the `return;`? I might be hurting some invariants by doing this...